client/src/components/graph/overlays/centroidLabels.js

```diff
@@ -57,7 +54,7 @@ class CentroidLabels extends PureComponent {
         fontWeight = "800";
       }
 
-      const selected = categoryValueSelected[categoryValueIndices.get(label)];
+      const selected = category.get(label) ?? true;
```

Comment:
this seems to imply if `category.get(label)` is `null` or `undefined`, we want to set `selected` to `true`. But in that case, what is selected?

Reply:
correct, the default state for missing values is selected (aka true).

all that is happening here is that our default selection state is true, and because we asynchronously load labels, we don't know all of them at reducer init time.  So if the reducer store does not yet "know" about the labels inside the data, we presume they are selected.

Reply:
Got it! Thanks for the contexts, Bruce 😄 Makes sense now!